src/pretalx/common/mixins/forms.py

```diff
@@ -87,6 +89,34 @@ class QuestionFieldsMixin:
     def get_field(self, *, question, initial, initial_object, readonly):
         from pretalx.submission.models import QuestionVariant
 
+        tz = pytz.timezone(question.event.timezone)
+
+        # Check (with question_required field) if the question should be required
+        if question.question_required == "require":
```

Comment:
Regarding this require/disable logic, I think this is not the best place to put it. What I'd like to see is a `required` property (or `cached_property`) on the `Question` class, so that we can keep using `question.required`. If that doesn't make sens to you, let me know and I can make the change myself.

Plus a similar property for `readonly`.

Reply:
✨  I added ```required``` property to ```Question``` class.